dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/WithSpanInterceptor.java

```diff
@@ -55,14 +58,22 @@ public WithSpanInterceptor() {
 
     @Inject
     public WithSpanInterceptor(final OpenTelemetry openTelemetry) {
-        InstrumenterBuilder<MethodRequest, Void> builder = Instrumenter.builder(openTelemetry, instrumentationName, new MethodRequestSpanNameExtractor());
-        MethodSpanAttributesExtractor<MethodRequest, Void> attributesExtractor = MethodSpanAttributesExtractor.newInstance(
+        Instrumenter<MethodRequest, Void> instrumenter = null;
+        try {
+            InstrumenterBuilder<MethodRequest, Void> builder = Instrumenter.builder(openTelemetry, instrumentationName, new MethodRequestSpanNameExtractor());
+            MethodSpanAttributesExtractor<MethodRequest, Void> attributesExtractor = MethodSpanAttributesExtractor.newInstance(
                                                                                                                            MethodRequest::getMethod,
                                                                                                                            new WithSpanParameterAttributeNamesExtractor(),
                                                                                                                            MethodRequest::getArgs);
 
-        this.instrumenter = builder.addAttributesExtractor(attributesExtractor)
+            instrumenter = builder.addAttributesExtractor(attributesExtractor)
                         .buildInstrumenter(methodRequest -> spanKindFromMethod(methodRequest.getMethod()));
+        } catch (Exception e) {
+            Tr.error(tc, Tr.formatMessage(tc, "CEMOT5101.telemetry.error", e));
+            instrumenter = null;
```

Comment:
Other parts of this class need updating to cope with `instrumenter` possibly being `null`.

Obviously it shouldn't happen, but if it did then we'd bombard the user with lots of `NullPointerException` messages.

Reply:
The only other ref to instrumenter is inside a try/catch already, I'm not sure what you mean by this.

Do you want an 
if(instrumenter == null) {
            Tr.error(tc, Tr.formatMessage(tc, "CEMOT5101.telemetry.error", e));
            return null;
 }
 
 To avoid an NPE?

Reply:
If this part fails, we don't want to log further exceptions later on which are as a result of the same problem.

If `instrumenter` is `null`, we don't want the `span()` method to try to use it.

I would change this part (line 95):
```java
boolean shouldStart = instrumenter.shouldStart(parentContext, methodRequest);
```

to this:
```java
boolean shouldStart = instrumenter != null && instrumenter.shouldStart(parentContext, methodRequest);
```

Reply:
Though seeing the comment further down this file, maybe we leave this file alone for the moment since it has other issues.